src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+@import 'variables';
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  font-family: $font-family;
+
+  &:hover {
+    transition-duration: $transition-time;
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    width: 166px;
+    height: 134px;
+    border-radius: 5px;
+    object-fit: contain;
+  }
+
+  &__title {
+    padding-top: 40px;
+    text-align: left;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $text-accent-color;
+  }
+
+  &:hover &__title {
+    transition-duration: $transition-time;
+    color: #34568b;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__review-count {
+    margin-top: 16px;
+    margin-left: 17px;
+    text-align: right;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
You can remove this, because 400 is default value in browser

Reply:
> You can remove this, because 400 is default value in browser

Not fixed without any reply